### DISCUSSION TITLE
Transformer for dropping NAs only

### DISCUSSION THREAD

**Discussion by TeoSkondras** [alan-turing-institute/sktime#4973] [Q&A] [answered]
Is there any transformer that can drop all the NA values in a column or in the entire dataframe, without having to modify it?

**Comment by fkiraly** [comment, marked as answer]
There is a generic `pandas` adapter which allows to add any `pandas` method as an `sktime` transformer step:

`from sktime.transformations.series.adapt import PandasTransformAdaptor`

In your case you would probably pick `method="dropna"`

**Reply by TeoSkondras** [reply]
Thank you, that worked!

**Reply by hliebert** [reply]
I've recently used this and found it to work only as the last transformer in a pipeline.

**Reply by fkiraly** [reply]
??? That sounds like a bug, @hliebert! Would be much appreciated if you could kindly report it, with a minimal failing example?

**Reply by hliebert** [reply]
Reported here: https://github.com/sktime/sktime/issues/4977. I wanted to reproduce it first, it only occured in conjunction with `IndexSubset`.

**Comment by TeoSkondras** [comment]
Another thing with PandasTransformAdaptor is that it does not support the invert_transform.
NotImplementedError: <class 'sktime.transformations.series.adapt.PandasTransformAdaptor'> does not implement inverse_transform

**Reply by fkiraly** [reply]
I thought it does? If the wrapped trafo has it, the `PandasTransformAdaptor` also has it.

Can you report an example?
